session/static.go

```diff
@@ -36,3 +37,16 @@ func OnSessionClose(f func(s Session)) {
 func CloseAll() {
 	DefaultSessionPool.CloseAll()
 }
+
+// New returns a new session instance
+// a NetworkEntity is a low-level network instance
+func New(entity networkentity.NetworkEntity, frontend bool, UID ...string) Session {
+	DefaultSessionPool = NewSessionPool()
```

Comment:
Is it intended that you are resetting the DefaultSessionPool every time you create a new session isntance?

Reply:
I'll remove this method, It doesn't make sense to be here I just used it for testing on Zooba